torch/testing/_internal/common_nn.py

```diff
@@ -3078,918 +3069,7 @@ def ctcloss_reference(log_probs, targets, input_lengths, target_lengths, blank=0
-    # dict(
-    #     module_name='CTCLoss',
-    #     desc='1d_target',
-    #     constructor_args=(14,),  # blank=14
-    #     extra_args=([50, 50, 50], [30, 25, 20]),  # input_lengths, target_lengths
-    #     input_fn=lambda: torch.randn(50, 3, 15).log_softmax(2),
-    #     target_fn=lambda: torch.randint(0, 14, (3, 30), dtype=torch.long),
-    #     reference_fn=lambda i, t, il, tl, m:
-    #         ctcloss_reference(i, t, il, tl, blank=14, reduction=get_reduction(m)),
-    #     check_sum_reduction=True,
-    #     check_gradgrad=False,
-    #     check_half=False,
-    # ),
-    dict(
-        module_name='CTCLoss',
-        desc='2d_int_target_lengths_intlists',
-        constructor_args=(0,),  # blank=0
-        extra_args=([50, 50, 50], [30, 25, 20]),  # input_lengths, target_lengths
-        input_fn=lambda: torch.randn(50, 3, 15).log_softmax(2),
-        target_fn=lambda: torch.randint(1, 15, (3, 30), dtype=torch.int),
-        reference_fn=lambda i, t, il, tl, m:
-            ctcloss_reference(i, t, il, tl, blank=0, reduction=get_reduction(m)),
-        check_forward_only=True,
-        check_sum_reduction=True,
-        check_gradgrad=False,
-        check_half=False,
-        # `CTCLoss` in C++ frontend doesn't accept integer list for `input_lengths` or `target_lengths`
-        test_cpp_api_parity=False,
-        check_jit=False,
-        default_dtype=torch.double,
-    ),
-    dict(
-        module_name='CTCLoss',
-        desc='2d_int_target_lengths_tensors',
-        constructor_args=(0,),  # blank=0
-        cpp_constructor_args='torch::nn::CTCLossOptions().blank(0)',
-        extra_args=(torch.tensor([50, 50, 50]), torch.tensor([30, 25, 20])),  # input_lengths, target_lengths
-        input_fn=lambda: torch.randn(50, 3, 15).log_softmax(2),
-        target_fn=lambda: torch.randint(1, 15, (3, 30), dtype=torch.int),
-        reference_fn=lambda i, t, il, tl, m:
-            ctcloss_reference(i, t, il, tl, blank=0, reduction=get_reduction(m)),
-        check_forward_only=True,
-        check_sum_reduction=True,
-        check_gradgrad=False,
-        check_half=False,
-        default_dtype=torch.double,
-    ),
-    dict(
-        module_name='CTCLoss',
-        desc='2d_lengths_tensors',
-        constructor_args=(0,),  # blank=0
-        cpp_constructor_args='torch::nn::CTCLossOptions().blank(0)',
-        extra_args=(torch.tensor([50, 50, 50]), torch.tensor([30, 25, 20])),  # input_lengths, target_lengths
-        input_fn=lambda: torch.randn(50, 3, 15).log_softmax(2),
-        target_fn=lambda: torch.randint(1, 15, (3, 30), dtype=torch.int),
-        reference_fn=lambda i, t, il, tl, m:
-            ctcloss_reference(i, t, il, tl, blank=0, reduction=get_reduction(m)),
-        check_forward_only=True,
-        check_sum_reduction=True,
-        check_gradgrad=False,
-        check_half=False,
-        default_dtype=torch.double,
-    ),
-]
+criterion_tests = []
```

Comment:
Is it important to still keep this around? Could we remove references to criterion_tests instead of having the empty list here?

Reply:
There are [a couple more tests being added to this list below](https://github.com/pytorch/pytorch/blob/b4defcb742e229e6343f3e21bab8b58d2ac1d8e4/torch/testing/_internal/common_nn.py#L3112-L3192) that weren't migrated in this PR

Reply:
ah i see!